users/templates/mail/welcome_and_next_steps.html

```diff
@@ -1,45 +1,67 @@
-                        your reference number</p>
-                    <p>We will be waiting your first payment before we add your account access in to the systems. Before this you can join our community meetings, open house and communication channels and ask if someone could help you. All Hacklab services work by voluntary basis so things might not work always. Please be patient. For the challenges help would be appreciated, and we would be more than happy for your contribution.</p>
-
-                    <h3>Bank account information is</h3>
-
-                    <p><b>Account holder</b>: {{settings.ACCOUNT_NAME}}<br />
-                        ACCOUNT_IBAN : {{settings.IBAN}}<br />
-                        ACCOUNT_BIC : {{settings.BIC}}<br />
-                        ref : {{user.reference_number}}</p>
-						
-			<p>Best regards Tampere Hacklab hackers!</p>
-
-                </td>
-            </tr>
-        </table>
-    </center>
-</body>
+      <table
+        width="100%"
+        border="0"
+        cellpadding="0"
+        cellspacing="0"
+        bgcolor="#F2F2F2"
+      >
+        <tr>
+          <td align="center" valign="top">
+            <h1>Welcome {{user.first_name}} to {{ settings.SITENAME }}</h1>
+
+            <p>
+              Great to have you onboard! Your membership application has been
+              approved and now is a great time to to join our community by
+              coming to the next community meeting on the first Tuesday of the
+              month or poping by weekly open house every Tuesday. It is also
+              highly recommendable to join our communication channels (Matrix,
+              Telegram or IRC and email list). Be happy to ask if you like to
+              know more!
+            </p>
+
+            <h2>Payment info</h2>
+            <p>
+              Your personal bank reference number is
+              <b>{{user.reference_number}}</b>. Remember always to use your
+              reference number.
+            </p>
+            <p>
+              We will be waiting your first payment before we add your account
+              access in to the systems. Before this you can join our community
+              meetings, open house and communication channels and ask if someone
+              could help you. All Hacklab services work by voluntary basis so
+              things might not work always. Please be patient. For the
+              challenges help would be appreciated, and we would be more than
+              happy for your contribution.
+            </p>
 
+            <h3>Bank account information is</h3>
+
+            <p>
+              <b>Account holder</b>: {{ settings.ACCOUNT_NAME }}<br />
+              ACCOUNT_IBAN : {{ settings.ACCOUNT_IBAN }}<br />
+              ACCOUNT_BIC : {{ settings.ACCOUNT_BIC }}<br />
+              ref : {{ user.reference_number }}
+            </p>
+
```

Comment:
Again, these should be generated dynamically from user's subscriptions.

I suggest to remove this section and just point a link to mulysa user page which already has all the details.

Reply:
bank information is always relevant

Reply:
Showing reference number may cause people to use it when they shouldn't.